user/views.py

```diff
@@ -206,3 +208,28 @@ def update_password(user, validated_data):
                 data={"message": "Old password is incorrect.", "code": 400},
                 status=status.HTTP_400_BAD_REQUEST,
             )
+
+
+class UserRoadmapDetailAPIView(RetrieveAPIView):
```

Comment:
Move to the roadmap app
Also you next pattern for view naming: {view_name}{generic_class}.
For instance: `UserRoadmapRetrieveAPIView`